src/Native/System.Native/pal_process.cpp

```diff
@@ -323,3 +329,74 @@ extern "C" int32_t WTermSig(int32_t status)
 {
     return WTERMSIG(status);
 }
+
+extern "C" int64_t PathConf(const char* path, PathConfName name)
+{
+    int32_t confValue = -1;
+    switch (name)
+    {
+        case PathConfName::PAL_PC_LINK_MAX:
+            confValue = _PC_LINK_MAX;
+            break;
+        case PathConfName::PAL_PC_MAX_CANON:
+            confValue = _PC_MAX_CANON;
+            break;
+        case PathConfName::PAL_PC_MAX_INPUT:
+            confValue = _PC_MAX_INPUT;
+            break;
+        case PathConfName::PAL_PC_NAME_MAX:
+            confValue = _PC_NAME_MAX;
+            break;
+        case PathConfName::PAL_PC_PATH_MAX:
+            confValue = _PC_PATH_MAX;
+            break;
+        case PathConfName::PAL_PC_PIPE_BUF:
+            confValue = _PC_PIPE_BUF;
+            break;
+        case PathConfName::PAL_PC_CHOWN_RESTRICTED:
+            confValue = _PC_CHOWN_RESTRICTED;
+            break;
+        case PathConfName::PAL_PC_NO_TRUNC:
+            confValue = _PC_NO_TRUNC;
+            break;
+        case PathConfName::PAL_PC_VDISABLE:
+            confValue = _PC_VDISABLE;
+            break;
+    }
+
+    if (confValue == -1)
+    {
+        assert(false && "Unknown PathConfName");
+        return EINVAL;
+    }
+
+    return pathconf(path, confValue);
+}
+
+extern "C" int64_t GetMaximumPath()
+{
+    int64_t result = pathconf("/", _PC_PATH_MAX);
+    if (result == -1)
+    {
+        result = PATH_MAX;
+    }
+
+    return result;
+}
+
+extern "C" int32_t GetPriority(PriorityWhich which, int32_t who)
+{
+    // GetPriority uses errno 0 to show succes to make sure we don't have a stale value
+    errno = 0;
+    return getpriority(static_cast<int>(which), static_cast<id_t>(who));
```

Comment:
Nope; it's more because id_t is declared as unsigned but the c# base class declares it as signed, so we need a conversion when sending back to the OS here